postgres/cloudpostgres/cloudpostgres.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cloudpostgres provides connections to managed PostgreSQL Cloud SQL instances.
+package cloudpostgres
+
+import (
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"errors"
+	"net"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/proxy"
+	"github.com/lib/pq"
+)
+
+// Params specifies how to connect to a Cloud SQL database.
+type Params struct {
+	ProjectID string
+	Region    string
+	Instance  string
```

Comment:
It might be worth linking to the documentation for instances: https://cloud.google.com/sql/docs/postgres/create-instance.

Reply:
I couldn't find a good enough link target that would describe the parameters here, so I added a link to the general docs in the package documentation. I think the example helps fill in the gap.